docs/releases/2.7.0.rst

```diff
@@ -0,0 +1,235 @@
+  - Different last activity messages for new translations and edits.
+  - Added landing page for anonymous users.
+  - The welcome page displays the top scorers over the last 30 days.
+  - User score is displayed in the header and dynamically updated when
+    translation actions are made.
+  - Several layout improvements that take into account different screen sizes.
+  - Filters allow sorting units according to their last action date.
+  - New navigation scheme:
+  
+    + All directories/files for a project are displayed in a new drop-down.
+      The differences between directories and files are visually highlighted.
+    + Directories/files can be navigated and translated across multiple
+      languages in a project.
+    + Tabs have been replaced in favor of drop-down menus.
+    + The editor search box is now displayed in the action links section,
+      keeping its positioning consistent with the overview page.
+    + A new action link in the editor, *Go back to overview*, allows users to
+      go back to the same place they entered translation mode from.
+
+  - Overview pages now report the last time a unit was added to a
+    store/project. In the browsing tables a *Last Updated* column is displayed
+    and in the extended stats *Created* and *Last Updated* dates.
+  - Implemented project notifications by reusing static pages. Notifications
+    are per-project and are displayed across languages (automatically adapting
+    any hyperlinks).
+  - Rewritten contact form. Allows both to contact site owners from any page as
+    well as to report any issues with strings.
+  - Revamped user profile pages. These now display user's latest activity and
+    its personal properties.
+  - Dropped support for old web browsers. Despite that Pootle might work on
+    such browsers. More on :ref:`Supported Browsers <browsers>`.
+  - Implemented export view for cross-language and cross-project views.
+  - Implemented global search. This allows to perform searches and edit units
+    in collections that span multiple projects across languages, multiple
+    languages across projects, or even the whole server.
+
+
+- Editor changes:
+
+  - Added keyboard shortcuts help.
+  - Several minor improvements.
+  - Placeables are used to highlight parts of the source text and easily insert
+    them into the translation.
+  - Added screenshot prefix URL for projects, which allow integrating
+    screenshots for units. The images are retrieved from public Evernote
+    notebooks.
+  - The editor now displays the numbering for units, not pages.
+  - When going through all units in the translation editor, users will be
+    automatically redirected back to overview.
+  - If the currently-submitted unit has pending checks, the editor won't
+    advance to the next unit and it will be updated displaying the unresolved
+    checks.
+  - Removed the concept of *pages* in the editor and its underlying APIs.  This
+    was problematic and buggy.
```

Comment:
Possible duplicate with "The editor now displays the numbering for units, not pages." above.